components/search/list-renderer/human-donor.js

```diff
@@ -31,6 +36,8 @@ export default function HumanDonor({ item: humanDonor, accessoryData }) {
         {lab && (
           <SearchListItemMeta>
             <div key="lab">{lab.title}</div>
+            <div key="collections">{collections}</div>
+            <div key="phynotypic-features">{phenotypicFeatures}</div>
```

Comment:
This won’t display collectinos or phenotypic features if the lab couldn’t be loaded, so you’ll probably want to put all three of these things in a non-sparse array and then display them in a loop inside `<SearchListItemMeta>`.